client/ayon_nuke/plugins/publish/validate_script_attributes.py

```diff
@@ -31,7 +31,12 @@ def process(self, instance):
 
         script_data = deepcopy(instance.context.data["scriptData"])
 
-        src_folder_attributes = instance.data["folderEntity"]["attrib"]
+        # Task may be optional for an instance
+        task_entity = instance.data.get("taskEntity")
+        if task_entity:
+            src_attributes = task_entity["attributes"]
```

Comment:
it is a dictionary or no?
```suggestion
            src_attributes = task_entity.get("attributes", {})
```

Reply:
Sorry, this was my bad. Typo.

Fixed with https://github.com/ynput/ayon-nuke/pull/13/commits/c5541d1f81c9fc89b3952c2743bad534b8175dda